welkin/models/patient.py

```diff
@@ -17,6 +20,14 @@ def delete(self):
     def __str__(self):
         return f"{self.firstName} {self.lastName}"
 
+    @property
+    def Encounter(self):
+        return self._client.Encounter(patient_id=self.id)
```

Comment:
If we continue to use the property decorator here, this should return an uninitialized class.

```python
@property
def Encounter(self):
    encounter = self._client.Encounter
    encounter.patient_id = self.id

    return encounter
```